odoo/fields.py

```diff
@@ -4245,9 +4245,17 @@ def convert_to_display_name(self, value, record):
     def get_depends(self, model):
         depends, depends_context = super().get_depends(model)
         if not self.compute and isinstance(self.domain, list):
+            try:
+                domain = self.get_domain_list(model)
+            except ValueError:
+                # "external ID not found" is raised during installation for callable domains
+                # ignore the domain in that case
+                if model.env.registry.ready:
```

Comment:
This is because some dependencies are built using a function like `lambda self: ... self.env.ref('mymodule.xxx')` which is not available until the module is installed.